src/utils/cookie.ts

```diff
@@ -21,6 +21,11 @@ interface Data {
 
 const cookies = new Cookies()
 
+const defaultOptions = {
+  path: '/',
+  httpOnly: true
+}
+
 export const setCookie = (name: string, value: string, option?: any) => {
   cookies.set(name, value, { ...option })
```

Comment:
```suggestion
  cookies.set(name, value, { path: '/', httpOnly: true, ...option })
```
이렇게 작성해도 동일 효과를 볼 수 있지 않을까요?!

Reply:
아하 어짜피 모든 쿠키는 저 속성을 따르니  그게 맞겠네요. 그나저나 httpOnly 속성을 넣으면 에러가 나서 적용을 할 수가 없네요. vercel에 질문을 올렸는데 아직 답을 못 받았어요 😥